examples/adaptive_library/BPMTriangularRotorNotches.py

```diff
@@ -242,12 +246,11 @@ def set_default_parameter(parameter_name, default_value):
 # * Go to the Geometry -> Radial tab to run the Adaptive Templates Script and display the new
 #   geometry
 
-
+# %%
+# .. note::
+#    When running in a Jupyter Notebook, the path for the Adaptive Templates script (PY file) must
+#    be given instead of ``sys.argv[0]`` when using the ``load_adaptive_script()`` method.
```

Comment:
```suggestion
#    When running in a Jupyter Notebook, you must provide the path for the Adaptive Templates script (PY file)
#    instead of ``sys.argv[0]`` when using the ``load_adaptive_script()`` method.
```